src/Codeception/Lib/InnerBrowser.php

```diff
@@ -1504,13 +1557,18 @@ public function grabValueFrom($field)
         $this->fail("Element $nodes is not a form field or does not contain a form field");
     }
 
+    /**
+     * @param string $name
+     * @param string|null $val
+     */
     public function setCookie($name, $val, array $params = [])
```

Comment:
Many public methods are documented in the interface: https://github.com/Codeception/Codeception/blob/4.1/src/Codeception/Lib/Interfaces/Web.php#L942-L958

I think that this docblock can replace that documentation in `build:docs`, so it would be better to avoid adding docbloks here.

Also, as we can see in https://codeception.com/docs/modules/PhpBrowser#setCookie ,
`@param` annotations are used by documentation generator,  which won't use type information for undocumented parameters.